stylesheets/_component.forms.scss

```diff
@@ -397,6 +397,7 @@ fieldset {
 .form__group.form__group .input-group .select2.select2,
 .form__group.form__group .input-group .form__control.form__control {
     flex: 1 1 auto;
+    overflow: hidden;
```

Comment:
Are you sure this is needed? In my quick tests the issue was fixed with `min-width: 0` and `overflow: hidden` on `.input-group-addon`

Reply:
select2s completely broke at small widths without this, I'll provide screenshot

Reply:
Before
![Screenshot 2019-04-30 at 10 15 14](https://user-images.githubusercontent.com/18653/56951896-dead1c80-6b30-11e9-9e0b-3c3a9678baca.png)

After
![Screenshot 2019-04-30 at 10 13 20](https://user-images.githubusercontent.com/18653/56951900-e10f7680-6b30-11e9-9367-e205d4571cab.png)

Reply:
👍